test/models/user_test.rb

```diff
@@ -207,7 +207,7 @@ def test_update_project_positions
   def test_find_and_activate_deferred_todos_that_are_ready
     assert_equal 1, @admin_user.deferred_todos.count
     @admin_user.deferred_todos[0].show_from = Time.now.utc - 5.seconds
-    @admin_user.deferred_todos[0].save
+    @admin_user.deferred_todos[0].save(:validate => false)
```

Comment:
Since we are setting the show_from date to something in the past the validation was not passing, so it was not saved with the updated date and the test was not doing what was supposed to do.